exercises/d2e1.js

```diff
@@ -10,4 +10,10 @@ const app = express()
  * It should respond with the simple string "pong"
  */
 
+app.get('/ping', (req, res) => {
+    res.send('pong');
+  });
```

Comment:
Sometimes, reviewers want to make a comment that is a _nitpick_. A _nitpick_ is a low priority feedback item that might pertain to matters of style, presentation, or readability but doesn't impact the functionality of the code in the slightest. It's usually left at the discretion of the author whether to rectify nitpicks. If you're short on time (who isn't?), you can disregard. Do note, though, that a lot of nitpicks can indicate a _[code smell](https://en.wikipedia.org/wiki/Code_smell)_, or failing to rectify sufficient numbers of nitpicks can introduce a code smell.

On that note...

Nit: Watch your use of white space. The closing brace `}` should be inline with the opening statement, as shown in suggestion here.
```suggestion
});
```